### PR TITLE
fix: disallow running immuadmin backup with current directory as source

### LINKED ISSUE #293: immudb backup does not work on windows

### PR DESCRIPTION
**Pull request by padurean**
closes #293

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ With immudb you can track changes in sensitive data in your transactional databa
 tamperproof immudb database. This allows you to keep an indelible history of sensitive data, for example debit/credit card transactions.
 <img align="right" src="img/immudb-mascot-small.png" width="256px"/>
 
-Traditional transaction logs are hard to scale and are mutable. So there is no way to know for sure if your data has been compromised.
+Traditional DB transactions and logs are hard to scale and are mutable. So there is no way to know for sure if your data has been compromised.
```
Comment (Gjergj) [inline README.md:18]: The original wording seems ok. It refers transaction log file on an ACID dbms.

File: cmd/immuadmin/command/backup.go [hunk 2]
```diff
@@ -189,6 +189,23 @@ func offlineBackup(src string, uncompressed bool, manualStopStart bool) (string,
...
+	}
+	srcAbs, err := filepath.Abs(src)
+	if err != nil {
+		return "", err
+	}
+	currDirAbs, err := filepath.Abs(currDir)
+	if err != nil {
+		return "", err
+	}
+	if srcAbs == currDirAbs {
```
Comment (Gjergj) [inline cmd/immuadmin/command/backup.go:204]: Perhaps the check can be done before the user is asked to authenticate.

File: README.md (hunk 1 above)
Comment (padurean) [reply README.md:18]: @Gjergj This change was specifically requested as such.

File: cmd/immuadmin/command/backup.go (hunk 2 above)
Comment (padurean) [reply cmd/immuadmin/command/backup.go]: Changed in the latest commit. 👍

**Pull request merged**